src/infinistore.cpp

```diff
@@ -71,6 +71,9 @@ struct Client {
     rdma_conn_info_t remote_info;
     rdma_conn_info_t local_info;
 
+    std::string key_to_check;
```

Comment:
is it OK to put these two into STACK? because these could be deserilized from recv_buf?

Reply:
I don't want to add another parameter in handle_request, we can do it when tcp proto is refined. 
or do you want me to process this request directly in on_read?

Reply:
we could do it later ;-)